2.6/root/usr/share/container-scripts/mongodb/init-functions.sh

```diff
@@ -0,0 +1,83 @@
+# mongo_create_admin creates the MongoDB admin user with password: MONGODB_ADMIN_PASSWORD
+# $1 - login parameters for mongo (optional)
+# $2 - host where to connect (localhost by default)
```

Comment:
No problem with this.

However in that shell coding style there is also documentation of parameters in function comment. So what is the OpenShift way?